.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: minimal
+
+branches:
+  only:
+    - master
+    - nightly-tests
+
+before_install:
+  # Decrypt secrets
+  - openssl aes-256-cbc -K $encrypted_36785d287aa1_key -iv $encrypted_36785d287aa1_iv
+    -in ci/keys/substra-208412-3be0df12d87a.json.enc -out ci/keys/substra-208412-3be0df12d87a.json
+    -d
+  - openssl aes-256-cbc -K $encrypted_3657ea9362f7_key -iv $encrypted_3657ea9362f7_iv
+    -in ci/keys/kaniko-secret.json.enc -out ci/keys/kaniko-secret.json -d
```

Comment:
Can you provide/add link to the travis documentation for this step :) ?

Reply:
+1 adding